Gemfile

```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 gem 'rouge'
 gem 'bourbon'
 gem 'redcarpet'
-gem 'jekyll'
+gem 'jekyll', '3.0.0.pre.beta2'
```

Comment:
Yes.I think we have a better sense of what to watch out for, plus some rough tests that will ensure our feed comes out an xml file and `robots` comes out as a `txt` file